src/textual/css/_error_tools.py

```diff
@@ -16,7 +16,7 @@ def friendly_list(words: Iterable[str], joiner: str = "or") -> str:
     Returns:
         str: List as prose.
     """
-    words = [repr(word) for word in sorted(words, key=str.lower)]
+    words = [repr(word) for word in sorted(words, key=str.lower) if word]
```

Comment:
Why the `if word`? Some times an empty string is a valid response.

Reply:
It just seemed weird to me, this function is only used for CSS units in error messages and it was strange to say a valid unit is `''`.

Reply:
Yeah, it's accurate, but doesn't read well.

Maybe we should handle it a different way, perhaps `valid values are "foo", "baz", or an empty string` ?

Reply:
The error messages all mention "an optional unit" then go on to say "valid units are a, b, c,...". I think saying `valid units are 'vw', 'vh', '%', or an empty string` doesn't sound great either.

Reply:
Fair enough. But lets make it optional to omit empty strings. I'd like to keep `friendly_list` generic enough to use in any context.